src/markdown_generators.js

```diff
@@ -77,8 +77,15 @@ const generateMarkdownRecursive = (page, courseUidsLookup, courseData) => {
   const hasParent = parents.length > 0
   const parent = hasParent ? parents[0] : null
   const inRootNav = page["parent_uid"] === courseData["uid"]
+  const isInstructorInsightsSection =
+    page["short_url"] === "instructor-insights" ||
+    (hasParent && parent["short_url"] === "instructor-insights")
+  const layout = isInstructorInsightsSection
+    ? "instructor_insights"
+    : "course_section"
```

Comment:
would it be relatively straightforward to add a test to make sure `"instructor_insights"` shows up when we expect it to?

Reply:
yep, working on that now